packages/subscriber/src/components/add-form/index.tsx

```diff
@@ -503,6 +510,13 @@ export const AddSubscriberForm: FunctionComponent< Props > = ( props ) => {
 					>
 						{ submitBtnName }
 					</NextButton>
+					{ props.showSkipLink && (
+						<div className="add-subscriber__form-skip-link-wrapper">
+							<button className="add-subscriber__form-skip-link" onClick={ props.onSkipBtnClick }>
```

Comment:
To be consistent, please take props constants at the beginning of the components and not access them directly from props. Everything is okay with this approach; I just want to follow the same style.

See: https://github.com/Automattic/wp-calypso/pull/77572/files#diff-313c3e20cc7156984e3eca7448261037ca4d5e5199f089e22c9f4ba2dba0ec05R55-R72

Reply:
Sure @bogiii - happy to update that!